src/listeners/s3-sqs-listener.ts

```diff
@@ -31,65 +30,58 @@ const s3 = new AWS.S3({
-  console.log('Starting import listener...');
+ // Connection URI
+ const dbUri = buildDatabaseUri(configManager.get("dbEndpoint"), configManager.get("dbUser"), configManager.get("dbPassword"), configManager.get("dbUriQuery"))
 
-  // Database Name
-  const dbName = configManager.get("dbName");
+ // Target bucket
+ const editorBucket = configManager.get("editorS3Bucket");
 
-  // Connection URI
-  const dbUri = buildDatabaseUri(configManager.get("dbEndpoint"), configManager.get("dbUser"), configManager.get("dbPassword"), configManager.get("dbUriQuery"))
+ // connect to cluster with TSL enabled 
+ const dbSSLValidate = configManager.get("dbSSLValidate");
+ const dbCertLocation = "/rds-combined-ca-bundle.pem";
 
-  // Target bucket
-  const editorBucket = configManager.get("editorS3Bucket");
+ let dbSSLCert: (string | Buffer)[] | undefined;
+ if(dbSSLValidate) {
+   dbSSLCert = [fs.readFileSync(dbCertLocation)]
+ }
+ const db = await initialiseDb(dbUri, dbName, dbSSLCert);
 
-  // connect to cluster with TSL enabled 
-  const dbSSLValidate = configManager.get("dbSSLValidate");
-  const dbCertLocation = "/rds-combined-ca-bundle.pem";
+ return importHandler(s3, db, editorBucket);
+}
 
-  let dbSSLCert: (string | Buffer)[] | undefined;
-  if(dbSSLValidate) {
-    dbSSLCert = [fs.readFileSync(dbCertLocation)]
-  }
-  const db = await initialiseDb(dbUri, dbName, dbSSLCert);
+export default async function start() {
+  console.log('Starting import listener...');
 
-  const handler = importHandler(s3, db, editorBucket);
+  const handler = await buildHandler();
 
   const S3SQSListener = Consumer.create({
     queueUrl: configManager.get("awsBucketInputEventQueueUrl"),
     region: configManager.get("awsRegion"),
     batchSize: 1,
     handleMessage: async (message) => {
-      /* istanbul ignore next */
-      const messageBody = JSON.parse(message.Body || "");
-      /* istanbul ignore next */
-      messageBody?.Records?.forEach((record: any) => {
-        console.log(
-          `SQS - AWS S3 uploaded event been consumed - { Key: ${record.s3.object.key}, Bucket: ${record.s3.bucket.name} }`
-        );
-      });
+      // Throw in here to leave message on queue
+      await handleMessage(handler.import, message)
     },
     sqs: new AWS.SQS({
       endpoint: configManager.get("awsEndpoint"),
     })
   });
-  S3SQSListener.on("message_received", async function(message) {
-    const messageBody = JSON.parse(message.Body);
-    if (messageBody?.Records?.length) {
-      messageBody.Records.forEach(async (record: any) => {
-        console.log(
-          `SQS - AWS S3 uploaded event been consumed - { Key: ${record.s3.object.key}, Bucket: ${record.s3.bucket.name} }`
-        );
-        try {
-          await handler.import(record.s3.object.key, record.s3.bucket.name);
-        } catch(error) {
-          S3SQSListener.emit('error', error);
-        }
-      });
-    }
+
+  S3SQSListener.on("message_processed", async function(message) {
```

Comment:
These event listeners should only be logging now, we might in future want to audit within these.